frontend/frontend-react/src/components/RackRow.js

```diff
@@ -57,21 +58,26 @@ export class RackRow extends Component {
         content =
           <td style={{
             fontSize: 7,
-            background: dispColor,
+            backgroundColor: dispColor,
             color: textColor,
             borderBottom: 'solid',
           }}>
+            <Link to={'/assets/' + this.props.id}>
             {this.props.model} {this.displayName()}
+            </Link>
```

Comment:
Any way to make sure that this text is contrasting? Pref the same colors that the text used to be? Like with a hyperlink but without the hyperlink coloring?

Reply:
yes done!